drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -54,6 +54,12 @@ matterManufacturer:
     vendorId: 0x141E
     productId: 0x0002
     deviceProfileName: contact-battery
+    # LeTianpai
```

Comment:
For readability, shouldn't the manufacturer information(# LeTianpai) go one space forward like others?

Reply:
Thanks for reviewing~, the issue has been addressed